libs/com.etendorx.das_core/src/main/java/com/etendorx/entities/mapper/lib/JsonPathEntityRetrieverBase.java

```diff
@@ -41,7 +44,7 @@ public E get(Object key) {
   public E get(TreeSet<String> keyValues) {
     Iterator<String> valueIterator = keyValues.iterator();
     if (keyValues.size() != getKeys().length) {
-      throw new IllegalArgumentException("Invalid number of key values");
+      throw new IllegalArgumentException("Mapping has misconfigured identifiers");
```

Comment:
**Suggestion:** The exception message in the get method has been changed. If the new message is less descriptive than the old one, consider reverting this change.
```suggestion
      throw new IllegalArgumentException("Invalid number of key values");
```